File: Java/src/main/java/com/lti/solvathon/dto/DependentAccountDTO.java
package com.lti.solvathon.dto;

import java.util.List;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
@AllArgsConstructor
@NoArgsConstructor
public class DependentAccountDTO {

	DependentSpendLimitDTO dependentSpendLimitDTO;
	List<MerchantHotlistDTO> merchantHotlistDTO;
}

File: PowerHouse/src/app/dto/dependentAccountDTO.ts
import { DependentSpendLimitDTO } from './DependentSpendLimitDTO';
import { MerchantHotlistDTO } from './merchantHotlistDTO';

export class DependentAccountDTO {
    dependentSpendLimitDTO!: DependentSpendLimitDTO
    merchantHotlistDTO!: MerchantHotlistDTO[]
}